src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -80,9 +80,13 @@ private protected virtual INameResolver CreateResolver()
             return null;
         }
 
-        /// <param name="localSymbols">An alternate config that can be provided. Should default to engine's config if null.</param>        
-        /// <returns></returns>
+        // Returns the INameResolver  and the corresponding Symbol table.         
         private protected INameResolver CreateResolverInternal(ReadOnlySymbolTable localSymbols = null)
+        {
+            return CreateResolverInternal(out _, localSymbols);
+        }
+
+        private protected INameResolver CreateResolverInternal(out ReadOnlySymbolTable symbols, ReadOnlySymbolTable localSymbols = null)
         {
```

Comment:
this is an awkward signature, returning the same thing it passes through the out param?

Reply:
It's private - so we have some flexibility here. 

If we go through line 100, it's not the same thing.